app/controllers/admin/projects_controller.rb

```diff
@@ -1,6 +1,6 @@
 class Admin::ProjectsController < Admin::BaseController
   def index
     @project = Project.new
-    @projects = Project.all
+    @projects = Project.all.order(id: :asc)
```

Comment:
remind me, does this sort all of the projects by id number so that the lowest id project is first?